src/include/stir/listmode/CListRecordROOT.h

```diff
@@ -127,6 +127,45 @@ class CListTimeROOT : public ListTime
     double timeB;
 };
 
+//! A class for storing and using an energy 'event' from a listmode file
+/*! \ingroup listmode
+ */
+class CListEnergyROOT : public ListEnergy
+{
+public:
+
+    void init_energy_from_data(double energy1, double energy2)
+    {
+        energyA = energy1;
+        energyB = energy2;
```

Comment:
As mentioned in #736, I cant figure out the benifit having `energyA` adn `energyB`.